bin/all_sky_search/pycbc_dtphase

```diff
@@ -60,7 +63,9 @@ parser.add_argument('--param-bin-dtype', default='int32',
                     choices=['int8', 'int16', 'int32'])
 args = parser.parse_args()
 
-assert len(args.relative_sensitivities) == len(args.ifos)
+if len(args.relative_sensitivities) != len(args.ifos):
+    parser.error('--relative-sensitivities requires one numerical argument '
+                 'for each detector')
```

Comment:
More a question than a comment - would making --relative-sensitivities a MultiDetOption be useful here?

Reply:
Is the benefit being able to do `--relative-sensitivities 1` to assume equally sensitive detectors?

Reply:
It would still have to be `--relative-sensitivities 1 1 1` (in a 3 ifo network), but I think that the advantage of being clearer in the input arguments, and not relying on the orders being the same would be an advantage

Reply:
I see `args.relative_sensitivities` is stored as an attribute in the output HDF file. Would changing that have any consequence downstream? Or I suppose I could keep the output format identical to what is now.

Reply:
good point - I think keeping it as it is should make sense then